components/post_markdown/post_markdown.jsx

```diff
@@ -53,6 +55,7 @@ export default class PostMarkdown extends React.PureComponent {
                 isRHS={this.props.isRHS}
                 message={this.props.message}
                 proxyImages={proxyImages}
+                searchTerm={this.props.options.searchTerm}
```

Comment:
What if we just pass along `options={this.props.options}` instead of pulling out the searchTerm and re-adding it to the options in the Markdown component